getting-things-gnome/gtg
# New file format (testing appreciated)

This is a branch for the new file format based on a spec from #279 with help from @johnnybubonic.
The spec is documented here: https://github.com/getting-things-gnome/gtg/blob/file_format_2/docs/contributors/file%20format.md

Testing super appreciated! GTG makes tons of backups, but this should still be as rock solid as possible.

closes #279 

## How to test:

Basically open GTG dev and play around. See if anything is saved incorrectly or there are any weird errors.
Another option is to use the import scripts from the `scripts` folder. You can import your tasks from GTG stable and test with them.
